### DISCUSSION TITLE
what is the purpose of the 'shear' transform

### DISCUSSION THREAD

**Discussion by seanmcd27** [vvvv/SVG#1099] [Q&A]
I understand what a shear transform is supposed to look like.  But, there is no transform named 'shear' in any version svg spec that I can find.  And, browsers and inkscape don't recognize this. Is the design intent here that the class internally convert to the appropriate raw 'matrix' transform and the implementation is incomplete?  If not, why is this in the library?

**Comment by mrbean-bremen** [comment]
This has been added in 2008 right after the repo had been moved from CodePlex to GitHub, referencing the old Microsoft issues. So unless somebody is around how remembers that transition, or who was developing SVG on the Microsoft side, I'm afraid it is difficult to find out now...
